### DISCUSSION TITLE
Joystick for LaserGRBL

### DISCUSSION THREAD

**Discussion by d1temnd** [arkypita/LaserGRBL#2445] [Ideas]
hello
I would like to control the machine using a PlayStation joystick. For example, somewhere in the settings, program key presses or joystick positions for certain functions or movements.

**Comment by Tribo-2dot0** [comment]
I made a video about this a while ago...

www.youtube.com/watch?v=_MIHQpxG0sg

**Comment by arkypita** [comment]
One of my collegue have made an extension/plugin that do what you need. I can ask him if he want to share the code

⁣Ottieni BlueMail per Android ​

Il giorno 8 dic 2024, 00:37, alle ore 00:37, Tribo-2dot0 ***@***.***> ha scritto:

**Reply by Tribo-2dot0** [reply]
If it's possible, I'd like to have this feature, it would make it easier without having to use another program as an intermediary.
